src/components/tests/assistant-code-block.test.tsx

```diff
@@ -263,4 +280,43 @@ describe( 'createCodeComponent', () => {
 			expect( getIpcApi().openLocalPath ).not.toHaveBeenCalled();
 		} );
 	} );
+
+	describe( 'when the "copy and open terminal" button is clicked', () => {
+		it( 'should not be visible for non-bash code blocks', () => {
+			render( <CodeBlock className="language-php" children="<?php echo 'Hello'; ?>" /> );
+
+			expect( screen.queryByText( 'Copy and open terminal' ) ).not.toBeInTheDocument();
+		} );
+
+		it( 'should be visible for bash code blocks', () => {
+			render( <CodeBlock className="language-bash" children="wp plugin list" /> );
+
+			expect( screen.getByText( 'Copy and open terminal' ) ).toBeInTheDocument();
+		} );
+
+		it( 'should be visible for sh code blocks', () => {
+			render( <CodeBlock className="language-sh" children="wp plugin list" /> );
+
+			expect( screen.getByText( 'Copy and open terminal' ) ).toBeInTheDocument();
```

Comment:
To check that the element is visible, we could use the expectation `toBeVisible`.